test/test_faker_company.rb

```diff
@@ -14,6 +14,10 @@ def test_duns_number
   end
 
   def test_logo
-	  assert @tester.logo.match(%r{http://pigment.github.io/fake-logos/logos/medium/color/\d+\.png})
+    assert @tester.logo.match(%r{http://pigment.github.io/fake-logos/logos/medium/color/\d+\.png})
+  end
+
+  def test_buzzword
+    assert @tester.buzzword.match(/\w+\.?/)
```

Comment:
Do you want to write something up?  I think it's a good idea to add to the contribution guide.  This strategy could also be waiting on one of the other features we're working on.